packages/fiori/src/Wizard.js

```diff
@@ -0,0 +1,650 @@
+ * The <code>ui5-wizard-step</code> provides the necessary API and it's up to the user of the component to use it to move to the next step.
+ * The user have to set its <code>selected</code> (and remove the <code>disabled</code> if set) property to true.
+ * <br>
+ *
+ * The component will automatically scroll to the content of the newly selected step.
+ * The Fiori 3 guidelines suggest having a button for moving to the next step upon click.
+ * The user can place such a button inside the <code>ui5-wizard-step</code>, for example after its content,
+ * and show/hide it when certain fields are filled or user defined criteria is met.
+ *
+ * <h3>Usage</h3>
+ * <h4>When to use:</h4>
+ * When the user has to accomplish a long set of tasks.
+ * <h4>When not to use:</h4>
+ * When the task has less than 3 steps.
+ *
+ * <h3>ES6 Module Import</h3>
+ * <code>import @ui5/webcomponents-fiori/dist/Wizard.js";</code> (includes <ui5-wizard-step>)
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.fiori.Wizard
+ * @extends UI5Element
+ * @tagname ui5-wizard
+ * @since 1.0.0-rc.10
+ * @appenddocs WizardStep
+ * @public
+ */
+class Wizard extends UI5Element {
+	constructor() {
+		super();
+
+		// Stores the scroll offsets of the steps,
+		// e.g. the steps' starting point.
+		this.stepScrollOffsets = [];
+
+		// Keeps track of the selected step index.
+		this.selectedStepIndex = 0;
+
+		// Indicates that selection will be changed
+		// due to user click.
+		this.selectionRequestedByClick = false;
+
+		// Indicates that selection will be changed
+		// due to user scroll.
+		this.selectionRequestedByScroll = false;
+
+		this.initItemNavigation();
+
+		this._onResize = this.onResize.bind(this);
+
+		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
+	}
+
+	static get metadata() {
+		return metadata;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get styles() {
+		return WizardCss;
+	}
+
+	static get template() {
+		return WizardTemplate;
+	}
+
+	static get dependencies() {
+		return [WizardTab, WizardStep];
+	}
+
+	static async onDefine() {
+		await fetchI18nBundle("@ui5/webcomponents-fiori");
+	}
+
+	static get PHONE_BREAKPOINT() {
+		return 559;
+	}
+
+	static get SCROLL_DEBOUNCE_RATE() {
+		return 25;
+	}
+
+	static get CONTENT_TOP_OFFSET() {
+		return 80;
+	}
+
+	onEnterDOM() {
+		ResizeHandler.register(this, this._onResize);
+	}
+
+	onExitDOM() {
+		ResizeHandler.deregister(this, this._onResize);
+	}
+
+	onBeforeRendering() {
+		this.syncSelection();
+	}
+
+	onAfterRendering() {
+		this.storeStepScrollOffsets();
+		this.scrollToSelectedStep();
+	}
+
+	/**
+	 * Normalizes the step selection as follows:
+	 * (1) If no selected is provided - the first step is going to be selected.
+	 * (2) If the selected steps are more than one - the first step is going to be selected.
+	 * (3) If the selected step is also disabled - the first step is going to be selected.
+	 * @private
+	 */
+	syncSelection() {
```

Comment:
I am not changing the disabled state anymore, but left the autoselection of the first in case of no selected steps and keeping the last from multiple selected steps. We can change it later, it is not even API.